### PR TITLE
custom installer: let user select wine settings

### PR DESCRIPTION
**Pull request by plata**
Would be great to have a bean to list the available options in a combobox.

### CODE REVIEW

File: Functions/QuickScript/InstallerScript/script.js [hunk 1]
```diff
@@ -28,6 +28,13 @@ InstallerScript.prototype.go = function() {
     // get installation file from concrete InstallerScript implementation
     var installationFile = this._installationFile(setupWizard);
 
+    // let user select wine settings if desired
+    if (this._wineUserSettings) {
+        this._wineArchitecture = setupWizard.textbox("Please enter the wine architecture.", this._wineArchitecture);
+        this._wineDistribution = setupWizard.textbox("Please enter the wine distribution.", this._wineDistribution);
```
Comment (qparis) [inline Functions/QuickScript/InstallerScript/script.js]: These should not be a textbox

Comment (plata) [inline Functions/QuickScript/InstallerScript/script.js]: Yes. Have a look at my comment in the PR.

**Comment by qparis** [comment]
We already have a lib to list available wine versions

File: Functions/QuickScript/InstallerScript/script.js (hunk 1 above)
Comment (qparis) [inline Functions/QuickScript/InstallerScript/script.js]: We already have a lib to list available wine versions :)

Comment (plata) [inline Functions/QuickScript/InstallerScript/script.js]: See it now...

Comment (qparis) [inline Functions/QuickScript/InstallerScript/script.js]: We have Wine.prototype.getAvailableVersions

Comment (plata) [inline Functions/QuickScript/InstallerScript/script.js]: Yes I found that as well now :)

**Pull request merged**